code/modules/modular_computers/hardware/portable_disk.dm

```diff
@@ -2,7 +2,7 @@
 	name = "data disk"
 	desc = "Removable disk used to store data."
 	power_usage = 10
-	icon_state = "datadisk6"
+	icon_state = "datadisk0"
```

Comment:
Is this an intended change? Looks undocumented, or am I missing something?

Reply:
It..I don't really know why, but travis was failing on this for some reason since apparently it wouldn't recognize this particular sprite, despite it being in the dmi it's trying to use. I probably should have asked about it.

Reply:
that's a bug with the automated test, feel free to ignore it, it comes and goes and won't be fixed by this

Reply:
fair enough. I'll just kick this back to what it was. (not that it really matters)